IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcFacilityPartCommon/Documentation.md

```diff
@@ -0,0 +1 @@
+A part of a facility.
```

Comment:
The IfcFacilityPartCommonTypeEnum enumeration existed because it listed the common facility part types (valid for rail, road etc). If this kind of facility part is now instantiated through this entity, maybe the documentation could be developed a bit further.
"A part of a facility. This kind of facility part may be included as part of any of the different concrete types of _IfcFacility_".